docs/articles/oso/how-to-open-source-software.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+How to open source software
+===========================
+
+.. tagslist:: howto
+
+When open sourcing source code there are some steps that needs to be completed.
+The purpose of these steps are to ensure that the code has a sound license, the
+source code repository is understandable and that there are clear descriptions
+on how to contribute. Following is a listing of these steps:
+
+#. Make sure that the following files are present in the root of the source code
+   repository:
+    * LICENSE: Containing the desired license for the code
+    * README.md: Containing information on where to find the documentation and
+      simple getting started info.
+    * CONTRIBUTING.md: Containing information on how to contribute to the
+      project and what rules there are.
+#. Go through file headers of all the source code and make sure that there is a
+   reference to the LICENSE file and an SPDX tag.
+#. Send a request to open source the code to the Open Source Office (OSO). The
+   request should contain an accessible link to the source code repository as
+   well as a statement of what the desired license is.
+#. The OSO makes sure that the desired license fits the business use cases.
+#. The OSO scans the code to make sure that it is compliant with the desired
+   license.
```

Comment:
Can we sync with the checklists we use internally?

Reply:
Yes we should